README.md

```diff
@@ -1,27 +1,33 @@
+# Dynamo
+
 ![mono_build](https://github.com/DynamoDS/Dynamo/workflows/mono_build/badge.svg)
 ![Dynamo-VS2019Build](https://github.com/DynamoDS/Dynamo/workflows/Dynamo-VS2019Build/badge.svg)
 
+[![Nuget](https://img.shields.io/nuget/v/DynamoVisualProgramming.Core?logo=nuget)](https://www.nuget.org/packages/DynamoVisualProgramming.Core)
+[![DynamoVisualProgramming.Core on fuget.org](https://www.fuget.org/packages/DynamoVisualProgramming.Core/badge.svg)](https://www.fuget.org/packages/DynamoVisualProgramming.Core)
```

Comment:
@avidit @mjkkirschner Is it fine for us to reference fuget on our readme?

Reply:
yes I think so besides it being down a bunch.